src/server/v2.0/handler/handler.go

```diff
@@ -14,6 +14,8 @@
 
 package handler
 
+// TODO move this file out of v2.0 folder as this is common for all versions of API
```

Comment:
Why this is common for all versions of APIs ?  

`restapi.HandlerAPI` and `restapi.Config` are both related to special API version.

Reply:
My bad, put it on the wrong file, the file that I said is `base.go`